.ci/azure-pipelines/build-ubuntu.yml

```diff
@@ -13,11 +13,21 @@ jobs:
     variables:
       BUILD_DIR: '$(Agent.BuildDirectory)/build'
       CMAKE_CXX_FLAGS: '-Wall -Wextra -Wabi -O2'
+      CCACHE_DIR: $(Pipeline.Workspace)/ccache
+      CCACHE_MAXSIZE: 15G
     steps:
+      - task: CacheBeta@0
+        inputs:
+          key: ccache | gcc | $(Agent.OS)
+          path: $(CCACHE_DIR)
+        displayName: ccache
```

Comment:
What about a bit more descriptive name, e.g. `Enable ccache` or `Download ccache files`? (Same for Windows steps.)